src/main/java/testapp/adapter/memory/InMemoryUserRepository.java

```diff
@@ -26,9 +25,9 @@ public InMemoryUserRepository() {
   }
 
   @Override
-  public User findUser(Integer userId) {
+  public User findUser(Integer facultyNumber) {
     for (User user : userList) {
-      if (user.getId().equals(userId)) {
+      if (user.getFacultyNumber().equals(facultyNumber)) {
```

Comment:
The product owner requirements where unclear on the subject, I understood that we will be using our app to serve only one univercity.